bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/AmazonEchoControlBindingConstants.java

```diff
@@ -115,9 +114,7 @@
     public static final String DEVICE_PROPERTY_SERIAL_NUMBER = "serialNumber";
     public static final String DEVICE_PROPERTY_FAMILY = "deviceFamily";
     public static final String DEVICE_PROPERTY_FLASH_BRIEFING_PROFILE = "configurationJson";
-    public static final String DEVICE_PROPERTY_APPLIANCE_ID = "applianceId";
-    public static final String DEVICE_PROPERTY_LIGHT_ENTITY_ID = "entityId";
-    public static final String DEVICE_PROPERTY_LIGHT_SUBDEVICE = "subDevice";
+    public static final String DEVICE_PROPERTY_ID = "id";

```

Comment:
The smartHomeDevice and smartHomeDeviceGroup have now just one property id. (applianceId)